### PR TITLE
fix #63608 m.hindustantimes.com

### LINKED ISSUE #63608: m.hindustantimes.com
### Issue URL (Annoyance)
 [https://m.hindustantimes.com/india-news/99k-india-nears-a-grim-landmark/story-xJQAVdrgUnCiuCQ0J05lUM.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fm.hindustantimes.com%2Findia-news%2F99k-india-nears-a-grim-landmark%2Fstory-xJQAVdrgUnCiuCQ0J05lUM.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Android 10
AdGuard product: | AdGuard for Android v3.5 (63)
Browser: | Samsung Browser
Stealth mode options: | Hide your search queries, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (180), <br/>Hide your Referrer from third-parties (), <br/>Hide your Referrer from third-parties
AdGuard mode: | VPN
Filtering quality: | High-quality
HTTPS filtering: | enabled
DNS requests blocking: | enabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> BarbBlock
Custom filter subscriptions: |

### PR DESCRIPTION
**Pull request by stanislav-atr**
#63608

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1521,6 +1521,7 @@ dmitripavlutin.com#?#article > div:has(> div > form[name="mc-embedded-subscribe-
 dmitripavlutin.com#?#main ~ aside > div > div:has(> form[name="mc-embedded-subscribe-form"])
 mtb-news.de##.mtbn-newsletter-bar
 kp.ru##.page-footer_subscribe
+m.hindustanttimes.com##.storyDetail > #loggedout
```
Comment (zloyden) [inline AnnoyancesFilter/sections/annoyances.txt:1524]: I can see it in desktop, so, better `hindustantimes.com##.storyArea > div.storyDetail + div.subscribe-update`

File: AnnoyancesFilter/sections/self-promo.txt [hunk 2]
```diff
@@ -627,6 +627,8 @@ m.blog.naver.com##.btm_banner
 wap.business-standard.com###popup123
 ko.dict.naver.com##.banner_vkorean_wrap
 nv.ua##.selfpromo_new
+m.hindustanttimes.com##.storyDetail > section[style^="clear: both;padding:0px 11px;"]
+m.hindustanttimes.com##div.mb20[style="text-align:center;"] > a > img
```
Comment (zloyden) [inline AnnoyancesFilter/sections/self-promo.txt:631]: Why not just `||m.hindustantimes.com/res/img/*/app.png`? And I guess it's for mobile-app section.
I don't see `m.hindustanttimes.com##.storyDetail > section[style^="clear: both;padding:0px 11px;"]`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/self-promo.txt (hunk 2 above)
Comment (stanislav-atr) [reply AnnoyancesFilter/sections/self-promo.txt:631]: paddin0px element:
screen

Comment (stanislav-atr) [reply AnnoyancesFilter/sections/self-promo.txt:631]: >And I guess it's for mobile-app section.
Mobile-app or mobile-specific? Looks like self-promo to me though.

File: AnnoyancesFilter/sections/self-promo.txt [hunk 3]
```diff
@@ -627,6 +627,8 @@ m.blog.naver.com##.btm_banner
 wap.business-standard.com###popup123
 ko.dict.naver.com##.banner_vkorean_wrap
 nv.ua##.selfpromo_new
+m.hindustanttimes.com##.storyDetail > section[style^="clear: both;padding:0px 11px;"]
+||m.hindustantimes.com/res/img/*/app.png
```
Comment (zloyden) [inline AnnoyancesFilter/sections/self-promo.txt:631]: Click on this element and you'll get a redirect to the Google Play.

Comment (stanislav-atr) [reply AnnoyancesFilter/sections/self-promo.txt:631]: Well, banner is placed on a website anyway?

Comment (zloyden) [reply AnnoyancesFilter/sections/self-promo.txt:631]: It's a mobile-app banner, you should move it to the correct section.

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 4]
```diff
@@ -53,6 +53,8 @@ moneycontrol.com##.app_btn_container
 nekto.me##.swal2-grow-fullscreen
 deeeep.io###play-store-container
 f2movies.to##a[href="/android-movies-apk"]
+m.hindustanttimes.com##.storyDetail > section[style^="clear: both;padding:0px 11px;"]
+||m.hindustantimes.com/res/img/*/app.png
```
Comment (zloyden) [inline AnnoyancesFilter/sections/mobile-app.txt:57]: I meant the last one. I'll move it manually.

**Review by zloyden — APPROVED** [review]

**Pull request merged**